make/dev.mk

```diff
@@ -25,10 +26,15 @@ dev-deploy-e2e-two-members: deploy-e2e-to-dev-namespaces-two-members print-reg-s
 .PHONY: deploy-e2e-to-dev-namespaces
 deploy-e2e-to-dev-namespaces:
 	$(MAKE) deploy-e2e MEMBER_NS=${DEV_MEMBER_NS} SECOND_MEMBER_MODE=false HOST_NS=${DEV_HOST_NS} REGISTRATION_SERVICE_NS=${DEV_REGISTRATION_SERVICE_NS} ENVIRONMENT=${DEV_ENVIRONMENT} E2E_TEST_EXECUTION=false IS_OSD=${IS_OSD} DEPLOY_LATEST=${DEPLOY_LATEST}
+	if [[ ${DEV_SSO} == true ]]; then \
+		$(MAKE) download-assets ASSETS_FOLDER=scripts/ci/dev-sso; \
+		$(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; \
+	fi
 
 .PHONY: deploy-e2e-to-dev-namespaces-two-members
 deploy-e2e-to-dev-namespaces-two-members:
 	$(MAKE) deploy-e2e MEMBER_NS=${DEV_MEMBER_NS} MEMBER_NS_2=${DEV_MEMBER_NS_2} HOST_NS=${DEV_HOST_NS} REGISTRATION_SERVICE_NS=${DEV_REGISTRATION_SERVICE_NS} ENVIRONMENT=${DEV_ENVIRONMENT} E2E_TEST_EXECUTION=false IS_OSD=${IS_OSD}
+	if [[ ${DEV_SSO} == true ]]; then $(MAKE) run-cicd-script SCRIPT_PATH=scripts/ci/setup-dev-sso.sh  SCRIPT_PARAMS="--sso-ns $(DEV_SSO_NS)"; fi
```

Comment:
don't you need to download the assets for this call and the other ones as well?

Reply:
+1 to Matous, but just for my understanding, what would be the main difference, if we don't download the assets Would we have any other way to use it ?

Reply:
Good point, I forgot to update the other make targets when I've moved the manifests to the cicd repo 🤦‍♂️ . 

Done in https://github.com/codeready-toolchain/toolchain-e2e/pull/996/commits/e4d36d5c134a7e564a83ce1f5c3e6c53d8c5bc31 Thanks!

Reply:
> but just for my understanding, what would be the main difference, if we don't download the assets Would we have any other way to use it ?

well in this case those targets will not work ( if they are executed before the one with the download assets call ) since the script will not have the manifests to apply , thus will error.